### PR TITLE
Disable key rotation

### PR DESCRIPTION
**Pull request by harshit777**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

File: ibm/service/kms/resource_ibm_kms_key_policies_test.go [hunk 1]
```diff
@@ -40,6 +40,38 @@ func TestAccIBMKMSKeyPolicy_basic_check(t *testing.T) {
 	})
 }
 
+func TestAccIBMKMSKeyPolicy_basic_check_enable(t *testing.T) {
+	instanceName := fmt.Sprintf("kms_%d", acctest.RandIntRange(10, 100))
+	keyName := fmt.Sprintf("key_%d", acctest.RandIntRange(10, 100))
+	rotationEnable := true
+	rotationBool := false
```
Comment (Sidd-kp) [inline ibm/service/kms/resource_ibm_kms_key_policies_test.go:47]:
```suggestion
	rotationDisable := false
```

**Review by Sidd-kp — APPROVED** [review]
Looks very neat. And the changes to the existing datasource looks great. Good job. The change suggested is optional. LGTM

**Comment by Sidd-kp** [comment]
Just one thing can you post test results also @harshit777

File: ibm/service/kms/resource_ibm_kms_key_policies_test.go (hunk 1 above)
Comment (harshit777) [reply ibm/service/kms/resource_ibm_kms_key_policies_test.go:47]: Thanks for the input, I did not changed bbecause rotationDisbale = false was a little misleading.

**Pull request closed without merging**